### PR TITLE
meremulatormodedialog: Fix compile error

### PR DESCRIPTION
**Pull request by sparkleholic**
meremulatormodedialog: Fix compile error

```
meremulatormodedialog.cpp:54:32:
error: invalid use of incomplete type ‘class QAction’
       m_action(new QAction(this)),
                                ^
In file included from /QT/5.11.1/gcc_64/include/QtGui/QList:1:0,
                 from ../../plugins/projectexplorer/abi.h:30,
                 from merdevice.h:26,
                 from meremulatordevice.h:26,
                 from meremulatormodedialog.h:36,
                 from meremulatormodedialog.cpp:30:
/Qt/5.11.1/gcc_64/include/QtGui/qevent.h:62:7:
note: forward declaration of ‘class QAction’
```

### CODE REVIEW

File: src/plugins/mer/meremulatormodedialog.h [hunk 1]
```diff
@@ -30,6 +30,7 @@
 #ifndef MEREMULATORMODEDIALOG_H
 #define MEREMULATORMODEDIALOG_H
 
+#include <QAction>
```
Comment (martyone) [inline src/plugins/mer/meremulatormodedialog.h]: This should go to the .cpp file unless I missed something.

Comment (sparkleholic) [reply src/plugins/mer/meremulatormodedialog.h]: Thanks for the quick review. I'll reflect the comment.

**Comment by martyone** [comment]
Please squash those commits to one, thank you.

**Comment by sparkleholic** [comment]
@martyone : I've squashed into one. Please check it, thanks.

**Comment by martyone** [comment]
lgtm, thanks

**Pull request merged**